andela/ah-titans-frontend
# #160011180  Create articles

#### What does this PR do?
Implement create article

#### Description of tasks to be completed
Create article

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout -b ft-create-articles-160011180
```
```sh
$ cd ah-titans-frontend
```

> Install the dependencies with the command below.

```sh
$ npm install
```
>Then run the app with the command below.
```sh
$ npm start
```
>Now you can navigate to your browser and you should be able to see a page as the one shown on the screenshot below.

### screenshots

Login to the app to view the red add button and click on it to go to the create article page

![image](https://user-images.githubusercontent.com/29063646/45481564-8e757b80-b754-11e8-912c-63089a0553ca.png)

-Then you will see the create article form as shown below

![image](https://user-images.githubusercontent.com/29063646/45351456-eaa69700-b5be-11e8-85dc-dfe6f6d2b411.png)

![image](https://user-images.githubusercontent.com/29063646/45351182-30af2b00-b5be-11e8-8ba9-71f024e1bb8e.png)

![image](https://user-images.githubusercontent.com/29063646/45351293-84217900-b5be-11e8-8e42-2f1045359d1b.png)

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160011180](https://www.pivotaltracker.com/story/show/160011180)
